README.md

```diff
@@ -1,117 +1,98 @@
-    * [Beginning with puppet_bolt_server](#beginning-with-puppet_bolt_server)
+1. [Installation - Step by step guide](#installation)
 1. [Usage - Configuration options and additional functionality](#usage)
 1. [Limitations - OS compatibility, etc.](#limitations)
-1. [Development - Guide for contributing to the module](#development)
 
 ## Description
 
-Briefly tell users why they might want to use your module. Explain what your
-module does and what kind of problems users can solve with it.
-
-This should be a fairly short description helps the user decide if your module
-is what they want.
+The goal of this module is to configure a dedicated Puppet Enterprise compiler to become a Bolt Server, with the intention of helping Orchestrator by taking the majority of the load when performing multiple concurrent plans runs. A compiler is ideal because it already has access to Puppet Enterprise, code manager and to its local PuppetDB.
 
 ## Setup
 
-### What puppet_bolt_server affects **OPTIONAL**
+### What puppet_bolt_server affects
 
-If it's obvious what your module touches, you can skip this section. For
-example, folks can probably figure out that your mysql_instance module affects
-their MySQL instances.
+The `puppet_bolt_server` module will perform the following activities:
 
-If there's more that they should know about, though, this is the place to
-mention:
+* Install Bolt in the server
+* Create the `/root/.puppetlabs/etc/bolt/bolt-defaults.yaml` file with custom configuration to:
+    * Use the PCP protocol
+    * Use the local PuppetDB
+    * Consume a Puppet token
 
-* Files, packages, services, or operations that the module will alter, impact,
-  or execute.
-* Dependencies that your module automatically installs.
-* Warnings or other important notices.
+## Installation
 
-### Setup Requirements **OPTIONAL**
+**Quickstart:** Configure the `puppet_bolt_server` using the PE Console
 
-If your module requires anything extra before setting up (pluginsync enabled,
-another module, etc.), mention it here.
+This setup will help you to quickly configure the `puppet_bolt_server` in your existing PE server.
 
-If your most recent release breaks compatibility or requires particular steps
-for upgrading, you might want to include an additional "Upgrading" section here.
+1. Add the [puppetlabs_puppet_bolt_server](https://github.com/puppetlabs/puppetlabs-puppet_bolt_server) to your control repo.
+1. Add a new Node Group from the PE Console
 
-### Beginning with puppet_bolt_server
-
-The very basic steps needed for a user to get the module up and running. This
-can include setup steps, if necessary, or it can be an example of the most basic
-use of the module.
+```
+  Parent name: PE Infrastructure
+  Group name: Bolt Server
+  Environment: production
+```
 
-## Usage
+1. Add the class `puppet_bolt_server` to the Bolt Sever group created in the step above.
+1. Add your dedicated compiler to the group using the Rules tab.
```

Comment:
Add your compiler _dedicated to running Bolt_ to the group using the Rules tab. Note that this compiler should not be in the compiler pool for catalog compilation.